### ISSUE TITLE
Undo stack is randomly cleared on non-Terminus views

### ISSUE THREAD

**Issue by willrowe** [randy3k/Terminus#313]
https://github.com/randy3k/Terminus/blob/fc0b132eae5a1118081bd97d167f53aa3c2c477f/terminus/commands.py#L489-L497

Due to this code, the undo stack in non-Terminus views will be randomly cleared. The conditional at the top is hard to parse initially, but it seems that the rest of the code in this method will only run if:
1. The view is *not* a widget __OR__
2. The view is a Terminus view

This means that any non-widget view will have its undo stack cleared.

**Comment by jfcherng** [comment]
Is `view.settings().get("is_widget", False) and \` necessary?

**Comment by willrowe** [comment]
You can have a terminal in a panel or a full view, so I am not sure if it is trying to only target one of those cases. Either way, greater care needs to be taken when calling `clear_undo_stack` to ensure that it only clearing Terminus views.

**Comment by randy3k** [comment]
Thanks for reporting it. A fix is on the wa.

**Comment by randy3k** [comment]
Fixed in v0.3.21 and it is on the way to Package Control.

**Comment by willrowe** [comment]
I'm not seeing a `v0.3.21`, but it looks like `v0.3.20` was removed so I uninstalled and reinstalled and `v0.3.19` was installed, effectively reverting the breaking code.

**Comment by randy3k** [comment]
Ya, I have removed v0.3.20 as it'll break a lot of people. `v0.3.21` should be ready soon from package control. (I forgot to push the tag to github)

**Comment by willrowe** [comment]
Thanks so much @randy3k! I can confirm it is no longer clearing the undo stack on non-Terminus views.

**Comment by randy3k** [comment]
I am sorry about the bug. Shouldn't make a release without testing at the first place.
